src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs

```diff
@@ -72,6 +90,23 @@ public HomeWorkspaceViewModel(HomeWorkspaceModel model, DynamoViewModel dynamoVi
             hwm.SetNodeDeltaState +=hwm_SetNodeDeltaState;
 
             dynamoViewModel.Model.ShutdownStarted += Model_ShutdownStarted;
+
+            SetupFooterNotificationItems();
+        }
+
+        /// <summary>
+        /// Setup the initial collection of FooterNotificationItems
+        /// </summary>
+        private void SetupFooterNotificationItems()
+        {
+            CurrentNotificationMessage = Properties.Resources.RunReady;   // Default value of the notification text block on opening
+
+            FooterNotificationItem [] footerItems = new FooterNotificationItem[2]; //TODO : change to 3 when Info is implemented
+
+            footerItems[0] = new FooterNotificationItem() { NotificationCount = 0, NotificationImage = "/DynamoCoreWpf;component/UI/Images/error.png", NotificationToolTip = "The number of Nodes in Error state."};
```

Comment:
So these are initialized every-time a new HomeWorkspace is created? Can we do this once for every Dynamo session.